ReleaseProcessAutomation/Jira/JiraCredentialManager.cs

```diff
@@ -0,0 +1,138 @@
+// Copyright (c) rubicon IT GmbH, www.rubicon.eu
+//
+// See the NOTICE file distributed with this work for additional information
+// regarding copyright ownership.  rubicon licenses this file to you under 
+// the Apache License, Version 2.0 (the "License"); you may not use this 
+// file except in compliance with the License.  You may obtain a copy of the 
+// License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software 
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT 
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the 
+// License for the specific language governing permissions and limitations
+// under the License.
+//
+
+using System;
+using System.Net;
+using AdysTech.CredentialManager;
+using ReleaseProcessAutomation.Jira.ServiceFacadeImplementations;
+using ReleaseProcessAutomation.ReadInput;
+using Serilog;
+using Spectre.Console;
+
+namespace ReleaseProcessAutomation.Jira;
+
+public interface IJiraCredentialManager
+{
+  Credentials GetCredential (string target);
+}
```

Comment:
I know that the other files with the steps always included the interface together in the same file. Usually we have them in separate file so for new code we should do that. I assume it was done with the steps to prevent having twice the files in the folder :D